conf/application.conf

```diff
@@ -354,6 +355,19 @@ feature {
   }
 }
 
+feature {
+  news {
+    nicSection {
+      short-description-en: "1.25 percentage points uplift in National Insurance contributions (base64 encoded)"
+      content-en: "<p>base64 encoded content with html</p>"
```

Comment:
Should the content not match what is already there? Or is it going to be replaced by a different app-config